### ISSUE TITLE
Bibls not displaying in html

### ISSUE THREAD

**Issue by wlpotter** [srophe/bethqatraye-data#2]
In the HTML, the cited bibls are not displaying properly. The footnote is displayed, but the accompanying bibliographic information is missing. 
See, for instance, https://bqgazetteer.bethmardutho.org/place/5541
which is citing: http://syriaca.org/bibl/Q7SV98GA

Is this an issue with the XSLT or with the new Bibl records that we added?

**Comment by wsalesky** [comment]
@wlpotter This is because the ptr is pointing to a remote server. 
@davidamichelson Is this a feature we will build in? Grabbing remote TEI records? If so, how do I differentiate between URI's, when to look for a remote TEI record?

**Comment by davidamichelson** [comment]
@wsalesky Thanks, please do add the following functionality for the BQG app (and future uses). The BQG app will not have its own bibliography but only rely on external bibls (either at Syriaca.org or elsewhere. Please have the following functionality:

1. Test URI for bibl
2. if Syriaca.org/bibl URI, grab the citation information from Syriaca.org in the same way that the Srophe App does it for Syriaca.org itself (unless this will slow things too much since in this case Syriaca.org is remote? in that case we could just treat the same as no. 3 below)
3. If not Syriaca.org bibl, use the citation information in the TEI record for the entry and construct a citation which includes the URL. 
Hypothetical example:

```
<bibl xml:id="bib5284-2">
<title level="m">Encyclopaedia of Islam, Second Edition</title>
 <editor>
   <name>
    <forename>P.</forename>
    <surname>Bearman</surname>
   </name>
  </editor>
<ptr target="http://dx.doi.org/10.1163/1573-3912_islam_COM_0251"/>
</bibl>
```
Rendered as prose in HTML:
P. Bearman, ed., _Encyclopaedia of Islam, Second Edition_, [http://dx.doi.org/10.1163/1573-3912_islam_COM_0251](http://dx.doi.org/10.1163/1573-3912_islam_COM_0251)

**Comment by wsalesky** [comment]
@wlpotter @davidamichelson Just to make sure we are all on the same page. 

For the present the Title and Author info will be added to the records and links will be generated to the Syriaca.org record.

**Comment by davidamichelson** [comment]
Yes, for now we will just insert the date manually into the place records. One question, how does the corpus handle bibliography? We will want to make it possible in the long run to make a call to the Syriaca.org server.

**Comment by wsalesky** [comment]
I don't think the corpus uses the bibl module. @jewalters?

**Comment by davidamichelson** [comment]
Need to check manually all bibls and produce a bibliography
